consensus/src/chained_bft/block_storage/mod.rs

```diff
@@ -30,6 +30,8 @@ pub enum VoteReceptionResult {
     NewQuorumCertificate(Arc<QuorumCert>),
     /// The vote completes a new TimeoutCertificate
     NewTimeoutCertificate(Arc<TimeoutCertificate>),
+    /// There might be some issues adding a vote
+    ErrorAddingVote,
```

Comment:
Can we return a `VerifyError` here? (this way we'd not lose the information about the verification problem)

Reply:
I was thinking that we could log the error before returning this, but if we log the error at the caller then this makes sense.

I'm still unsure on how to log this error  though :D